### PR TITLE
Replaced deprecated sample code in proxy-setup docs

### LINKED ISSUE #3139: Deprecated code in proxy-setups document
In the document of [proxy-setup](http://flask.pocoo.org/docs/1.0/deploying/wsgi-standalone/#proxy-setups),
`from werkzeug.contrib.fixers import ProxyFix` has been deprecated since werkzeug 0.15.
The sample code should be `from werkzeug.middleware.proxy_fix import ProxyFix`.

The sample usage become `app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1)` that 
works for my application behind HTTPS proxy.

Reference:
* https://werkzeug.palletsprojects.com/en/0.15.x/contrib/fixers/
* https://werkzeug.palletsprojects.com/en/0.15.x/middleware/proxy_fix

### PR DESCRIPTION
**Pull request by Tom-McDonald**
Altered the proxy-setup docs to replace deprecated sample code as per issue #3139 

It's my first PR so if there's anything I've done wrong then give me a shout!

closes #3139

### CODE REVIEW

File: docs/deploying/wsgi-standalone.rst [hunk 1]
```diff
@@ -123,8 +123,8 @@ If your httpd is not providing these headers, the most common setup invokes the
 host being set from ``X-Forwarded-Host`` and the remote address from
 ``X-Forwarded-For``::
 
-    from werkzeug.contrib.fixers import ProxyFix
-    app.wsgi_app = ProxyFix(app.wsgi_app)
+    from werkzeug.middleware.proxy_fix import ProxyFix
+    app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1)
```
Comment (davidism) [inline docs/deploying/wsgi-standalone.rst]: Also need to set `x_host=1` based on the server config above.

Comment (Tom-McDonald) [reply docs/deploying/wsgi-standalone.rst]: Thanks for pointing out, I'll add that in

Comment (linnil1) [reply docs/deploying/wsgi-standalone.rst]: I found that `middleware.proxy_fix` is only appeared above 0.15, so I think document may contains WARNING for werkzeug version.

Comment (Tom-McDonald) [reply docs/deploying/wsgi-standalone.rst]: @linnil1 okay so you reckon we should add in a warning just below the sample usage that it only works above 0.15? Is it worth documenting the previous usage alongside that or is there no point?

Comment (linnil1) [reply docs/deploying/wsgi-standalone.rst]: I have no idea for adding it or not.
Please ask @davidism.

Comment (Tom-McDonald) [reply docs/deploying/wsgi-standalone.rst]: Is this okay to merge @davidism or do you reckon anything else needs adding/changing?

**Pull request merged**